lib/mayaUsd/ufe/Global.cpp

```diff
@@ -117,6 +117,9 @@ void mayaStartWaitCursor()
 
 void mayaStopWaitCursor() { MGlobal::executeCommand("waitCursor -state 0"); }
 
+// Note: MayaUsd::ufe::getStage takes two parameters, so wrap it in a function taking only one.
+PXR_NS::UsdStageWeakPtr mayaGetStage(const Ufe::Path& path) { return MayaUsd::ufe::getStage(path); }
```

Comment:
Is the issue with the default parameters that requires this wrap?

Reply:
yes